src/check.h.in

```diff
@@ -690,7 +699,7 @@ CK_DLL_EXP void CK_EXPORT _ck_assert_failed(const char *file, int line,
  *
  * @since 0.9.6
  */
```

Comment:
There may need to be additional documentation on these macros, stating that NULL is allowed for *_eq and *_ne, but if a pointer is NULL for any other macro the macro will report a test failure.